aptos-move/e2e-tests/src/executor.rs

```diff
@@ -489,9 +489,10 @@ impl FakeExecutor {
         ).map(BlockOutput::into_transaction_outputs_forced)
     }
 
-    pub fn execute_transaction_block(
+    pub fn execute_transaction_block_with_resolver(
         &self,
         txn_block: Vec<Transaction>,
+        resolver: &(impl StateView + Sync),
```

Comment:
Why? executor has already the reference to data store, and now we pass it another resolver?

Reply:
This is for online execution where the code state depends on fake data store while other data state depends on DebuggerStateView